ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicationManager.java

```diff
@@ -79,6 +77,12 @@ public VcrReplicationManager(VerifiableProperties properties, CloudConfig cloudC
     this.cloudStorageCompactor =
         cloudConfig.cloudBlobCompactionEnabled ? new CloudStorageCompactor(cloudDestination, cloudConfig,
             partitionToPartitionInfo.keySet(), vcrMetrics) : null;
+    // We need a datacenter to replicate from. For DR, we currently deploy VCR nodes on same datacenter as ambry-server.
+    // For serving data from cloud, we must specify the ambry-server datacenter(s) from which to replicate.
+    if (!cloudConfig.cloudIsVcr && cloudConfig.vcrCrossColoReplicationPeerDatacenters.isEmpty()) {
+      throw new IllegalStateException(
+          "Either this should be a vcr replicator OR one or more vcr cross colo replication peer datacenter should be specified");
```

Comment:
Simplify to "One or more VCR cross colo replication peer datacenter should be specified"

Reply:
ok